utah-geological-survey.csl

```diff
@@ -135,9 +135,9 @@
           <group prefix=", ">
             <text term="in" font-style="italic"/>
             <names variable="editor translator" prefix=" " suffix="," delimiter=", ">
-              <name and="text" sort-separator=", " initialize-with="." name-as-sort-order="all"/>
+              <name and="text" sort-separator=", " initialize-with="." name-as-sort-order="all" delimiter-precedes-last="always"/>
               <et-al term="and others"/>
-              <label form="short" prefix=" "/>
+              <label prefix=","/>
```

Comment:
Did you test this?

To get "in Miller, D.M., Todd, V.R., and Howard, K.A., editors" (p. 20), you'd probably need `<label prefix=", "/>`, no? (with added space)